internal/etos/suitestarter/suitestarter.go

```diff
@@ -320,7 +345,7 @@ func (r *ETOSSuiteStarterDeployment) reconcileRolebinding(ctx context.Context, l
 		if !apierrors.IsNotFound(err) {
 			return rolebinding, err
 		}
-		logger.Info("Creating a new role binding for the suite starter")
+		logger.Info(fmt.Sprintf("Creating a new role binding for the suite starter: %s", roleName))
```

Comment:
```suggestion
		logger.Infof("Creating a new role binding for the suite starter: %s", roleName)
```

Reply:
logr.Logger does not have .Infof()

Reply:
Ah right. Sorry. Should be:

```suggestion
		logger.Info("Creating a new role binding for the suite starter", "roleName", roleName)
```